dnf-docker-test/features/steps/repo_utils.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/python
+
+import os
+import re
+import gzip
+import xml.dom.minidom
+import ConfigParser
```

Comment:
this will fail on python3 (code must be compatible with both versions of python)